ionic_user_interface/src/views/ForgotPassword.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <ion-page>
+    <ion-content :fullscreen="true">
+      <ion-grid>
+        <ion-row color="primary" class="ion-align-items-center ion-justify-content-center">
+          <ion-col class="ion-align-self-center" size-md="6" size-lg="5" size-xs="12">
+            <div class="ion-text-center">
+              <h1>Forgot Password</h1>
+            </div>
+            <div class="ion-padding ion-text-center">
+              <MessageBox ref="msgBox" :color="msgBoxColor" class="rounded margin" />
```

Comment:
If you're not gonna display a non-danger message, there's no need for the msgBoxColor ref, you can just put `color="danger"` and remove the code logic to do with msgBoxColor